packages/jest-snapshot/src/__tests__/__snapshots__/printDiffOrStringified.test.ts.snap

```diff
@@ -212,30 +253,33 @@ exports[`multi line small changes 1`] = `
 `;
 
 exports[`single line large changes 1`] = `
-Snapshot: <g>"<i>A</i>rray length<i> must be a finite positive integer</i>"</>
-Received: <r>"<i>Invalid a</i>rray length"</>
+<g>- Snapshot  1 -</>
+<r>+ Received  1 +</>
+
+<g>- <i>A</i>rray length<i> must be a finite positive integer</i></>
+<r>+ <i>Invalid a</i>rray length</>
 `;
 
 exports[`without serialize backtick single line expected and multi line received 1`] = `
-<g>- Snapshot</>
-<r>+ Received</>
+<g>- Snapshot  1 -</>
+<r>+ Received  2 +</>
```

Comment:
Not that I can think of a better way to display it, but `+ Snapshot [n] -` is confusing to me at a glance.

I wonder if we're better off long-form:

`+ Snapshot [n] lines added` (or something like that but more accurate)

Reply:
My submission: `+ Snapshot +[n] line(s)`

Reply:
I agree that a postfix sign is confusing. Who knows what I was thinking.

Here is a picture with prefix sign and `line(s)` at the left versus without at the right:

<img width="800" alt="counts with and without lines" src="https://user-images.githubusercontent.com/11862657/65701869-11136500-e050-11e9-91be-d42c6fd2f026.png">

I think that after reviewing a few snapshots, people will see numbers alone more quickly.

If we improve the other matchers, then counts in annotation lines will be like:

```text
- Expected  - 2
+ Received  + 2
```

Reply:
This way either would work I think, clear enough without the text as well